examples/lie_labs.py

```diff
@@ -0,0 +1,87 @@
+import torch
+
+import theseus.labs.lie as lie
+import theseus.labs.lie.functional as lieF
+
+batch_size = 5
+
+# ### Lie Tensor creation functions
+g1 = lie.SE3.rand(batch_size, requires_grad=True)
+print(f"Created SE3 tensor with shape {g1.shape}")
+g2 = g1.clone()
+
+# Can create from a tensor as long as it's consistent with the desired ltype
+g3_data = lieF.so3.rand(5)
```

Comment:
I just wanted to make sure that I understand this API correctly: 

typically `so3` implies the lie algebra elements which lie in the tangent space, and `SO3` implies LieGroup elements on the manifold, and therefore I was expecting this API to simply return random 3-vectors. 

However, this API returns 3x3 matrices of tensor type. If this is expected `lieF.so3.rand` should simply be a function like `lieF.SO3rand` or something similar to follow functional style as well.

Reply:
Hi @akashsharma02. Thanks for pointing this out, I can see why this is confusing. We were aware of this convention, but, tbh, I forgot we have `so3` name under `LieF`; I've been mostly focusing on the `LieTensor` class lately. 

Your suggestion makes sense, although it would also need to be applied to all functions under `lie.so3` (see [here](https://github.com/facebookresearch/theseus/blob/main/theseus/labs/lie/functional/lie_group.py#L173) and [here](https://github.com/facebookresearch/theseus/blob/main/theseus/labs/lie/functional/so3_impl.py#L920)). Any thoughts on something like `lieF.SO3.op_name()` instead? 

cc @fantaosha @mhmukadam

Reply:
> Any thoughts on something like `lieF.SO3.op_name()` instead?
> 

I like `lieF.SO3.op_name()` as it makes it consistent with `lie.SO3`.
Sophus also uses functional form operations such as `sp.SO3.exp()`, so this API wouldn't be new to a lot of people.

Reply:
Thanks @akashsharma02. Agreed with @luisenp we can switch to `lieF.SO3.op_name()`.

I am wondering do you see the utility in keeping the functional api exposed via `lieF` or do you find it redundant?

Reply:
@luisenp  I think the `lieF.SO3.op_name()` would work well. 

@mhmukadam I do think there is utility in exposing both APIs, just from the standpoint of allowing users to choose either functional style or OOP style. In either case, some functions like `rand` *will* need to be exposed from the functional side.

Reply:
I agree that `SO3` and `SE3` are better. BTW, I wonder if we need to change `adjoint->Adjoint` since `Adjoint` and `adjoint` have different meaning for Lie group operators.